test/article/model.js

```diff
@@ -4,8 +4,8 @@
 var should = require('should'),
     app = require('../../server'),
     mongoose = require('mongoose'),
-    User = mongoose.model('User'),
-    Article = mongoose.model('Article');
+    User = mongoose.model('User');
+    //  Article = mongoose.model('Article');
```

Comment:
we should totally avoid including commented out code in source control

Reply:
Thanks for the feedback, I was trying to ensure my test passed on travis but I see now that was wrong. I have fixed that in my latest commit.